project/Projects.scala

```diff
@@ -24,10 +24,10 @@ object Projects extends Build {
 
   lazy val kamon = Project("kamon", file("."))
     .enablePlugins(CrossPerProjectPlugin)
-    .aggregate(kamonCore, kamonScala, kamonAkka, kamonSpray, kamonNewrelic, kamonPlayground, kamonTestkit,
-      kamonStatsD, kamonRiemann, kamonDatadog, kamonSPM, kamonSystemMetrics, kamonLogReporter, kamonAkkaRemote, kamonJdbc, kamonElasticsearch,
-      kamonAnnotation, kamonPlay23, kamonPlay24, kamonPlay25, kamonJMXReporter, kamonFluentd, kamonKhronus,
-      kamonAutoweave, kamonInfluxDB)
+    .aggregate(kamonCore, kamonScala, kamonAkka, /*kamonSpray, kamonNewrelic,*/ kamonPlayground, kamonTestkit,
+      kamonStatsD, kamonRiemann, kamonDatadog, /*kamonSPM,*/ kamonSystemMetrics, kamonLogReporter, /*kamonAkkaRemote,*/ kamonJdbc, kamonElasticsearch,
+      kamonAnnotation, /*kamonPlay23, kamonPlay24, kamonPlay25,*/ kamonJMXReporter, /*kamonFluentd,*/ kamonKhronus,
+      kamonAutoweave/*, kamonInfluxDB*/)
```

Comment:
This just removes the projects that have missing 2.12 deps (or in the case of kamonAkkaRemote won't compile).

Haven't worked out a proper way to make this list depend on the current scala version.